contrib/tracing/log_utxo_coins.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+
+import sys
+from bcc import BPF, USDT
+from bcc.utils import printb
+
+""" Example script to log details about add/spend/uncache coin events.
+utilizing USDT probes and the utxo:coins_added, utxo:coins_spent,
+utxo:coins_uncached  tracepoint. """
+
+program = """
+# include <uapi/linux/ptrace.h>
+struct coin_data
+{
+  u32 height;
+  long value;
+  u32 isCoinBase;
+  u64 usage;
+};
+
+
+BPF_ARRAY(data_arr, struct coin_data, 1);
```

Comment:
I don't think you need a `BPF_ARRAY` here. This is only used when we can't fit the struct into the 512 byte eBPF stack. See [doc/tracing.md#ebpf-virtual-machine-limits](https://github.com/bitcoin/bitcoin/blob/master/doc/tracing.md#ebpf-virtual-machine-limits).

See e.g. [this](https://github.com/bitcoin/bitcoin/blob/92f3a4b4d0531976c9484f5b5075487e0925b619/contrib/tracing/p2p_monitor.py#L45-L54) from the P2P monitor as an example.